hatgame-backend/server/message/message.go

```diff
@@ -0,0 +1,21 @@
+package message
```

Comment:
Since game has to send messages on events, game.go imports message. But server.go imports game. If i move message on the same level as server.go it will be in the server package and that will result in an import cycle (which I'm not sure how to resolve). Also if I had to create an Error message in game this will be written as server.Error and I just thought that message,Error reads better that's why I separated it into a message package.

Reply:
Done